tests/lib/azure/fixture/fixture.go

```diff
@@ -70,15 +72,15 @@ func Run(
 		logger, teardown := testlog.New(t, testname)
 		defer teardown()
 
-		session := azure.NewSession(t)
+		session := NewSession(t)
 		resgroup := fmt.Sprintf("klb-test-fixture-%s-%d", testname, rand.Intn(9999999))
 
-		resources := azure.NewResourceGroup(ctx, t, session, logger)
+		resources := NewResourceGroup(ctx, t, session, logger)
 		defer func() {
 			// We cant use an expired context when cleaning up state from Azure.
 			ctx, cancel := context.WithTimeout(context.Background(), resourceCleanupTimeout)
 			defer cancel()
-			resources := azure.NewResourceGroup(ctx, t, session, logger)
+			resources := NewResourceGroup(ctx, t, session, logger)
 			resources.Delete(t, resgroup)
```

Comment:
vnet does not have a delete method anymore, we are not testing deletion (it was not working anyway :-)